labgrid/remote/client.py

```diff
@@ -936,9 +938,17 @@ def sd_mux(self):
                 except NoDriverFoundError:
                     drv = USBSDMuxDriver(target, name=None)
                 break
+            elif isinstance(resource, NetworkUSBSDWireDevice):
```

Comment:
No need for an `elif` here.
```suggestion
            if isinstance(resource, NetworkUSBSDWireDevice):
```